napari/layers/_markers_layer/view/properties.py

```diff
@@ -89,7 +89,7 @@ def changeEdgeColor(self, text):
         self.layer.edge_color = text
 
     def changeSymbol(self, text):
-        self.layer.symbol = text
+        self.layer.symbol = text.upper()
```

Comment:
is this necessary since `upper` will be called on it anyways when it gets converted to an Enum?

Reply:
Ah good catch!